### ISSUE TITLE
a little problem about Dropout

### ISSUE THREAD

**Issue by hanyaqian** [lvapeab/nmt-keras#24]
Hi @lvapeab 

>  Dropout consists in randomly setting a fraction `p` of input units to 0 at each update during training time.

I have understood above. But why to divide the remaining result by 1-p. describe  [here](https://github.com/MarcBS/keras/blob/master/keras/backend/theano_backend.py#L1839)

Regards

**Comment by lvapeab** [comment]
Hi,
maybe this link is useful for you:
https://github.com/fchollet/keras/issues/3305https://github.com/fchollet/keras/issues/3305

Regards!

**Comment by hanyaqian** [comment]
Thanks for your reply.
